sceptre/stack.py

```diff
@@ -618,15 +618,28 @@ def _format_parameters(self, parameters):
         for name, value in parameters.items():
             if value is None:
                 continue
-            if isinstance(value, list):
-                value = ",".join(value)
             formatted_parameters.append({
                 "ParameterKey": name,
-                "ParameterValue": value
+                "ParameterValue": self._format_value(value)
             })
 
         return formatted_parameters
 
+    def _format_value(self, value):
+        """
+        Converts CloudFormation value to the format used by Boto3.
+        """
+        if isinstance(value, list):
+            # recurse
+            new_list = [self._format_value(item) for item in value]
+            return ",".join(new_list)
+        if value is True:
```

Comment:
Why do we want to downcase this?

Reply:
If your parameter is defined like this:
```yaml
Parameters:
  Key:
    Type: String
    AllowedValues:
      - False
      - True
```
Only `"false"`/`"true"` will work, not `"False"`/`"True"`. Same goes for `False`/`True`. So you need to downcase it.

Reply:
Makes sense